src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepository.scala

```diff
@@ -20,15 +20,15 @@ class FileMetadataRepository(db: Database)(implicit val executionContext: Execut
   private val insertFileStatusQuery =
```

Comment:
`InsertFileStatusQuery` doesn't seem to be used. The class also has a few unused imports that can be removed.